conf/messages

```diff
@@ -55,6 +55,24 @@ checkingRecords.ffid=Identifying file formats
 fileChecksResults.header=Record check results
 fileChecksResults.title=Record check results
 fileChecksResults.description=Here are your results
+transferSummary.header=Transfer Summary
+transferSummary.title=Transfer Summary
+transferSummary.seriesReference=Series reference
+transferSummary.consignmentReference=Consignment reference
+transferSummary.transferringBody=Transferring body
+transferSummary.printableReport=Printable Report
```

Comment:
Should "report" be lower case for consistency: `Printable report`

Reply:
I copied the wireframe, but yes, I agree, it probably should.